CHANGES.md

```diff
@@ -1,3 +1,77 @@
+## stable-2.6.0
+
+This release introduces experimental distributed tracing support, adds request
+and response headers to `linkerd tap`, dramatically improves the performance of
+the dashboard on large clusters, adds traffic split visualizations to the
+dashboard, adds a public Helm repo, and many more improvements!
+
+For more details, see the announcement blog post: TODO
+https://linkerd.io/2019/08/20/announcing-linkerd-2.6/
+
+To install this release, run: `curl https://run.linkerd.io/install | sh`
+
+**Upgrade notes**: Use the `linkerd upgrade` command to upgrade the control
```

Comment:
Does this release also require rolling the data plane? cc @ihcsim

Reply:
Yes, it does. I think we should just link this to the upgrade notes in the docs. See https://github.com/linkerd/website/pull/547. Once the PR is merged, the official link should be https://linkerd.io/2/tasks/upgrade/#upgrade-notice-stable-2-6-0.